### PR TITLE
fix: update ChaosEngine crd manifest file in graphql server

### PR DESCRIPTION
**Pull request by namkyu1999**
## Proposed changes

Currently, the litmus-graphql-server uses an old CRD manifest file. So when you execute the experiment in Litmus version 3.0.0-beta12, you can see the error logs below.

`{"mainLogs":"W1003 08:59:55.273647       1 client_config.go:552] Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.\n2023/10/03 08:59:55 Error Creating Resource : ChaosEngine.litmuschaos.io 'pod-network-loss-h6srhrls' is invalid: [spec.experiments[0].spec.probe[0].runProperties.interval: Invalid value: 'string': spec.experiments[0].spec.probe[0].runProperties.interval in body must be of type integer: 'string', spec.experiments[0].spec.probe[0].runProperties.probeTimeout: Invalid value: 'string': spec.experiments[0].spec.probe[0].runProperties.probeTimeout in body must be of type integer: 'string']\n"}`

In this PR, I updated ChaosEngine manifest file.
I used this file (https://github.com/litmuschaos/chaos-operator/blob/master/deploy/crds/chaosengine_crd.yaml)

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by amityt — APPROVED** [review]

**Comment by Saranya-jena** [comment]
@namkyu1999 did you try applying this crd: https://github.com/litmuschaos/litmus/blob/master/mkdocs/docs/3.0.0-beta12/litmus-portal-crds-3.0.0-beta12.yml ?

**Comment by namkyu1999** [comment]
> @namkyu1999 did you try applying this crd: https://github.com/litmuschaos/litmus/blob/master/mkdocs/docs/3.0.0-beta12/litmus-portal-crds-3.0.0-beta12.yml ?

Hi @Saranya-jena, As you said, if we apply that crd file to all target clusters, it works fine, because once the crd is defined, even if we update to an old manifest in the future, the crd won't change. But if we download the control plane and don't download the crd as you should, the graphql server will create a manifest using the old crd file, so we need that pr.

**Review by gdsoumya — APPROVED** [review]

**Comment by namkyu1999** [comment]
Some variable types changed in this pr https://github.com/litmuschaos/chaos-operator/pull/459. But the crd version not changed. So anyone who already applied old version of ChaosEngine CRD will face the same problem.

**Review by neelanjan00 — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Review by Saranya-jena — APPROVED** [review]

File: chaoscenter/graphql/server/manifests/cluster/2a_litmus_crds.yaml [hunk 1]
```diff
@@ -1216,116 +1231,116 @@ spec:
...
+                                                description: 'HostPath represents a pre-existing
                                                 file or directory on the host machine
                                                 that is directly exposed to the container.
                                                 This is generally used for system agents
                                                 or other privileged things that are
                                                 allowed to see the host machine. Most
                                                 containers will NOT need this. More
                                                 info: https://kubernetes.io/docs/concepts/storage/volumes#hostpath
-                                                # TODO(jonesdl) We need to restrict
+                                                --- TODO(jonesdl) We need to restrict
```
[the remark this answers is not in the document]
Comment (namkyu1999) [reply chaoscenter/graphql/server/manifests/cluster/2a_litmus_crds.yaml]: Thanks for the comments, I updated it :)

**Review by ispeakc0de — APPROVED** [review]

**Pull request merged**